synapse/config/key.py

```diff
@@ -95,9 +95,49 @@ def read_config(self, config, config_dir_path, **kwargs):
 
         # if neither trusted_key_servers nor perspectives are given, use the default.
         if "perspectives" not in config and "trusted_key_servers" not in config:
+            logger.warn(
+                """
+                Synapse requires that a list of trusted key servers are
```

Comment:
won't this be oddly formatted, with the narrow columns and all of the left-padding? I suggest extracting constants to the top of the file.